cmake/modules/Findlilv.cmake

```diff
@@ -84,5 +86,12 @@ if(lilv_FOUND)
         INTERFACE_COMPILE_OPTIONS "${PC_lilv_CFLAGS_OTHER}"
         INTERFACE_INCLUDE_DIRECTORIES "${lilv_INCLUDE_DIR}"
     )
+    is_static_library(lilv_IS_STATIC lilv::lilv)
+    if(lilv_IS_STATIC)
+      find_package(sord CONFIG REQUIRED)
```

Comment:
```suggestion
      find_package(sord REQUIRED)
```

Reply:
As already mentioned, this is how it is called in the lilv CMakeList.txt both place need to be use the same method.

Reply:
This means this CMake module can *only* work with vcpkg because the upstream library does not ship a CMake config module.

Reply:
@Holzhaus what are your thoughts on this?

Reply:
Ping @Holzhaus, this is all that is left on this PR.

Reply:
I tend to agree with @Be-ing. We shouldn't hardcode config mode if we don't need to. vcpkg usage should make it easier to get the dependencies but not be a strict requirement IMHO.

How about setting [`CMAKE_FIND_PACKAGE_PREFER_CONFIG`](https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_PACKAGE_PREFER_CONFIG.html) instead and falling back to module mode if vcpkg is not used and hence no cmake config is available?

Reply:
> How about setting CMAKE_FIND_PACKAGE_PREFER_CONFIG instead? This will automatically fall back to module mode if vcpkg is not used and hence no cmake config is available.

How about we take that discussion to Zulip so we can move on with this PR?

Reply:
I thought this was an issue with this PR that prevents the PR from being merged and that you wanted my opinion on? If not, I don't get why you pinged me here ;-)

Reply:
We can remove the `CONFIG` without deciding on whether to use `CMAKE_FIND_PACKAGE_PREFER_CONFIG`.

Reply:
Sure, but this will only work if the system does not provide a find module for this lib.

Reply:
We are here in the static build path, that is currently only used for vcpkg builds with macOS. Vcpkg used config mode explicite for some reasons. If we remove it we will have module mode. I guess they want to ensure that the config file is used. We want the same.

We are discussing here a future case, that will be instantly notable because of a not found config message. So I think it is here a good idea to follow the vcpkg approach and fix the future issue in future.

Reply:
I think vcpkg uses CONFIG in its own modules because it needs that to reference its own modules rather than the application's.

Reply:
We want the same.

Reply:
The context here is different. This module might be copied into another application which doesn't use vcpkg and has its own Findsord.cmake module. Hardcoding CONFIG would break that. I don't think there's a good reason to make this code less flexible by doing that.

Reply:
I don't care about writing the module to use it in other applications, sorry.

Reply:
It is more about the general principle not to hardcode unnecessary assumptions. Anyway I am done bikeshedding this. If anyone really needs to change it, it would simply be deleting a single word.

Reply:
I have asked about [upstreaming these CMake build scripts](https://gitlab.com/lv2/lilv/-/issues/6).